client/components/shared/StarAverage.jsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 
 const StarAverage = () => {
   return (
-    <div className="star-container">
+    <span className="star-container">
```

Comment:
I thought we had discussed that this component was in my purview so I'm surprised to see changes to this file in your pull request?